source/renderer/app/stores/WalletMigrationStore.js

```diff
@@ -106,6 +107,21 @@ export default class WalletMigrationStore extends Store {
   getExportedWalletByIndex = (index: number): ?ExportedByronWallet =>
     this.exportedWallets.find(w => w.index === index);
 
+  @action _initiateMigration = () => {
+    this.walletMigrationStep = 0;
```

Comment:
@DeeJayElly shouldn't we use the variables you declared here instead of integers? https://github.com/input-output-hk/daedalus/pull/2014/files#diff-2ee5a0431d36653b94842124ac703db9R9

Reply:
Yeah i will refactor this

Reply:
Fixed